react/LazyRender.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FunctionComponent, useRef, useState } from 'react'
```

Comment:
This just wraps the lazy rendering hook into a component for easier handling.

copied from https://github.com/vtex-apps/render-runtime/blob/master/react/components/LazyRender.tsx